`Microsoft.NET.Workload.Emscripten.Manifest-6.0.400` nuget is missing. I don't see a flow PR for this either.

`
  /tmp/install-workload/o11ldwim.qvg/restore/Restore.csproj : error NU1101: Unable to find package Microsoft.NET.Workload.Emscripten.Manifest-6.0.400. No packages exist with this id in source(s): artifacts-nonshipping-packages, artifacts-shipping-packages, darc-pub-dotnet-emsdk-52e9452, dotnet-eng, dotnet-public, dotnet-tools, dotnet6, dotnet6-transport, richnav
`

--- a/NuGet.config
+++ b/NuGet.config
@@ -9,7 +9,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
-    <add key="darc-pub-dotnet-emsdk-52e9452" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-52e9452f/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-1026ad5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-1026ad55/nuget/v3/index.json" />
     <!--  End: Package sources from dotnet-emsdk -->
     <!--  Begin: Package sources from dotnet-wcf -->
     <!--  End: Package sources from dotnet-wcf -->

--- a/eng/Version.Details.xml
+++ b/eng/Version.Details.xml
@@ -20,6 +20,10 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>1026ad55ddb8587bf68829d3f5e8abafa65548fe</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.400" Version="6.0.8">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>1026ad55ddb8587bf68829d3f5e8abafa65548fe</Sha>
+    </Dependency>
     <Dependency Name="System.ServiceModel.Primitives" Version="4.9.0">
       <Uri>https://github.com/dotnet/wcf</Uri>
       <Sha>04c3656e0325277aefa18378f24238d39b259222</Sha>

--- a/eng/Version.Details.xml
+++ b/eng/Version.Details.xml
@@ -8,17 +8,17 @@
       <Uri>https://github.com/dotnet/msquic</Uri>
       <Sha>98129287d56a5e0348c291ce4260e630b4aa510d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.100" Version="6.0.4">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.100" Version="6.0.8">
       <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>52e9452f82e26f9fcae791e84c082ae22f1ef66f</Sha>
+      <Sha>1026ad55ddb8587bf68829d3f5e8abafa65548fe</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.200" Version="6.0.4">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.200" Version="6.0.8">
       <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>52e9452f82e26f9fcae791e84c082ae22f1ef66f</Sha>
+      <Sha>1026ad55ddb8587bf68829d3f5e8abafa65548fe</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.300" Version="6.0.4">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Manifest-6.0.300" Version="6.0.8">
       <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>52e9452f82e26f9fcae791e84c082ae22f1ef66f</Sha>
+      <Sha>1026ad55ddb8587bf68829d3f5e8abafa65548fe</Sha>
     </Dependency>
     <Dependency Name="System.ServiceModel.Primitives" Version="4.9.0">
       <Uri>https://github.com/dotnet/wcf</Uri>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -170,9 +170,9 @@
     <runtimeosx1012x64MicrosoftNETCoreRuntimeMonoLLVMSdkVersion>11.1.0-alpha.1.21416.1</runtimeosx1012x64MicrosoftNETCoreRuntimeMonoLLVMSdkVersion>
     <runtimeosx1012x64MicrosoftNETCoreRuntimeMonoLLVMToolsVersion>11.1.0-alpha.1.21416.1</runtimeosx1012x64MicrosoftNETCoreRuntimeMonoLLVMToolsVersion>
     <!-- emscripten / Node -->
-    <MicrosoftNETWorkloadEmscriptenManifest60100Version>6.0.4</MicrosoftNETWorkloadEmscriptenManifest60100Version>
-    <MicrosoftNETWorkloadEmscriptenManifest60200Version>6.0.4</MicrosoftNETWorkloadEmscriptenManifest60200Version>
-    <MicrosoftNETWorkloadEmscriptenManifest60300Version>6.0.4</MicrosoftNETWorkloadEmscriptenManifest60300Version>
+    <MicrosoftNETWorkloadEmscriptenManifest60100Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60100Version>
+    <MicrosoftNETWorkloadEmscriptenManifest60200Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60200Version>
+    <MicrosoftNETWorkloadEmscriptenManifest60300Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60300Version>
     <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETWorkloadEmscriptenManifest60100Version)</MicrosoftNETRuntimeEmscriptenVersion>
     <!-- workloads -->
     <SwixPackageVersion>1.1.87-gba258badda</SwixPackageVersion>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -150,7 +150,7 @@
     <SQLitePCLRawbundle_greenVersion>2.0.4</SQLitePCLRawbundle_greenVersion>
     <MoqVersion>4.12.0</MoqVersion>
     <FsCheckVersion>2.14.3</FsCheckVersion>
-    <SdkVersionForWorkloadTesting>6.0.300-preview.22159.1</SdkVersionForWorkloadTesting>
+    <SdkVersionForWorkloadTesting>6.0.400</SdkVersionForWorkloadTesting>
     <!-- Docs -->
     <MicrosoftPrivateIntellisenseVersion>6.0.0-preview-20211019.1</MicrosoftPrivateIntellisenseVersion>
     <!-- ILLink -->
@@ -173,7 +173,9 @@
     <MicrosoftNETWorkloadEmscriptenManifest60100Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60100Version>
     <MicrosoftNETWorkloadEmscriptenManifest60200Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60200Version>
     <MicrosoftNETWorkloadEmscriptenManifest60300Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60300Version>
+    <MicrosoftNETWorkloadEmscriptenManifest60400Version>6.0.8</MicrosoftNETWorkloadEmscriptenManifest60400Version>
     <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETWorkloadEmscriptenManifest60100Version)</MicrosoftNETRuntimeEmscriptenVersion>
+
     <!-- workloads -->
     <SwixPackageVersion>1.1.87-gba258badda</SwixPackageVersion>
     <WixPackageVersion>1.0.0-v3.14.0.5722</WixPackageVersion>

--- a/src/tasks/WorkloadBuildTasks/InstallWorkloadFromArtifacts.cs
+++ b/src/tasks/WorkloadBuildTasks/InstallWorkloadFromArtifacts.cs
@@ -4,9 +4,9 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
 using System.IO;
 using System.Linq;
-using System.Text;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using Microsoft.Build.Framework;
@@ -94,11 +94,18 @@ namespace Microsoft.Workload.Build.Tasks
         {
             Log.LogMessage(MessageImportance.High, $"Installing workload manifest for {name}/{version} for sdk band {VersionBand}");
 
+            string sdkManifestsVersionBadDir = Path.Combine(SdkDir, "sdk-manifests", VersionBand);
+            if (!Directory.Exists(sdkManifestsVersionBadDir))
+            {
+                Log.LogMessage(MessageImportance.Low, $"Creating directory {sdkManifestsVersionBadDir} because it does not already exist.");
+                Directory.CreateDirectory(sdkManifestsVersionBadDir);
+            }
+
             // Find any existing directory with the manifest name, ignoring the case
             // Multiple directories for a manifest, differing only in case causes
             // workload install to fail due to duplicate manifests!
             // This is applicable only on case-sensitive filesystems
-            string outputDir = FindSubDirIgnoringCase(Path.Combine(SdkDir, "sdk-manifests", VersionBand), name);
+            string outputDir = FindOrCreateSubDirIgnoringCase(sdkManifestsVersionBadDir, name);
 
             PackageReference pkgRef = new(Name: $"{name}.Manifest-{VersionBand}",
                                           Version: version,
@@ -171,7 +178,7 @@ namespace Microsoft.Workload.Build.Tasks
             return false;
         }
 
-        private string FindSubDirIgnoringCase(string parentDir, string dirName)
+        private string FindOrCreateSubDirIgnoringCase(string parentDir, string dirName)
         {
             IEnumerable<string> matchingDirs = Directory.EnumerateDirectories(parentDir,
                                                             dirName,
@@ -184,7 +191,7 @@ namespace Microsoft.Workload.Build.Tasks
                                 + $"{Environment.NewLine}Using the first one: {first}");
             }
 
-            return first ?? Path.Combine(parentDir, dirName);
+            return first ?? Path.Combine(parentDir, dirName.ToLower(CultureInfo.InvariantCulture));
         }
 
         private record ManifestInformation(

